source/slang/syntax-base-defs.h

```diff
@@ -60,6 +49,15 @@ ABSTRACT_SYNTAX_CLASS(Val, NodeBase)
     )
 END_SYNTAX_CLASS()
 
+RAW(
```

Comment:
Abusing `RAW(...)` like this is a little odd (the assumption is that `RAW` always holds stuff that belongs inside the enclosing class declaration), but since it works in practice and all that macro hackery is disgusting beyond belief, I'm fine with leaving it.

Reply:
Agreed it is not a thing of beauty. Because of the mechanism of how 'as' works I have to guarantee the free functions are available if the type is - and this was the easiest way to do so.